src/Native/LibTorchSharp/THSTensor.cpp

```diff
@@ -1771,9 +1771,9 @@ void THSTensor_topk(const Tensor tensor, Tensor* (*allocator)(size_t length), co
 {
     CATCH(
         auto topk = tensor->topk(k, dim, largest, sorted);
-    Tensor * result = allocator(2);
-    result[0] = new torch::Tensor(std::get<0>(topk));
-    result[1] = new torch::Tensor(std::get<1>(topk));
+        Tensor * result = allocator(2);
```

Comment:
Style nit: just looking at pointer use above, the style seems to be to not have a space before the asterisk, e.g.
```suggestion
        Tensor* result = allocator(2);
```

Reply:
K. Will fix.